examples/src/main/kotlin/example/ForwardingRefs.kt

```diff
@@ -1,11 +1,12 @@
 package example
 
+import kotlinx.js.jso
 import org.w3c.dom.HTMLInputElement
 import react.*
 import react.dom.html.ReactHTML.input
 
 external interface Focusable {
-    fun focus()
+    var focus: () -> Unit
```

Comment:
I don see the purpose for these changes, `Focusable` it is not props, it is immutable object, it is output data from components

Reply:
@turansky if you want to discuss it, let's move these changes to separate PR

Reply:
`Focusable` is some kind of custom object, it is not declared is react types of in typescript typings in browser, what kind of symmetric are we talking about?

Reply:
It was decided to discuss these changes later.